eventlogger/src/main/java/gov/cms/ab2d/eventlogger/eventloggers/sql/SqlEventLogger.java

```diff
@@ -4,30 +4,45 @@
 import gov.cms.ab2d.eventlogger.EventLoggingException;
 import gov.cms.ab2d.eventlogger.LoggableEvent;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Propagation;
 import org.springframework.transaction.annotation.Transactional;
 
 @Service
 @Slf4j
 public class SqlEventLogger implements EventLogger {
+    @Value("${execution.env:dev}")
+    private String appEnv;
+
     private final SqlMapperConfig mapperConfig;
+    private final JdbcTemplate template;
 
-    public SqlEventLogger(SqlMapperConfig mapperConfig) {
+    public SqlEventLogger(SqlMapperConfig mapperConfig, JdbcTemplate template) {
         this.mapperConfig = mapperConfig;
+        this.template = template;
     }
 
     @Override
     @Transactional(propagation = Propagation.REQUIRES_NEW)
     public void log(LoggableEvent event) {
+        event.setEnvironment(appEnv);
         try {
             SqlEventMapper mapper = mapperConfig.getMapper(event.getClass());
             if (mapper == null) {
-                throw new EventLoggingException("Can't find logger for " + event.getClass().toString());
+                throw new EventLoggingException("Can't find SQL logger for " + event.getClass().toString());
             }
             mapper.log(event);
         } catch (Exception ex) {
             log.error("Error in logging event " + event.toString());
         }
     }
+
+    public void updateAwsId(String awsId, LoggableEvent event) {
```

Comment:
Not sure if awsId is being used in this method? It looks like the aws_id is the sql column name though

Reply:
Okay, I changed it to use the parameter and not assume that it is in the object.